tests/etl/test_load_artifacts.py

```diff
@@ -73,7 +74,7 @@ def test_load_non_ascii_textlog_errors(test_job):
     assert TextLogError.objects.get(line_number=1588).line == "07:51:29  WARNING - <U+01D400>"
 
 
-def _add_job_summary(job, expected_new):
+def _add_job_summary(job, expected_new, date=None):
```

Comment:
Second argument is output-ish while third one is input-ish. Please switch positions.

Reply:
easy fix, updated this last commit to address this!  thanks for the feedback and deploying to prototype.